### ISSUE TITLE
White swings before windfury imbued weapon and multi windfury procs without internal shared cooldown

### ISSUE THREAD

**Issue by Delinquent1** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3734]
After patch 14.15.2 implemented on 26/12/2015 i noticed some strange double white hits occurring exactly before the windfury imbue proc.

![](http://i.imgur.com/LBydWZM.jpg)

So i decided to take a closer look into this so i got a 2h weapon, imbued it with windfury and opened the combatlog and i saw the case is the same.
There is always white swing in-before windfury imbue proc.
 
![](http://i.imgur.com/4Zwxwvg.jpg?1)

 
I really wish you look into this issue more carefully since there should not be white attack in the same swing before the windfury.
The event is on successful swing it is either white attack or windfury.
This causes again unpleasant environment in pvp mostly due to the fact in 1 global 1 enhancement can deal stormstrike>double white hits (if dual wield)>windfury in 1 global
Something i show in the image below but instead of oh wf proc mh...
Give or take this is around 4,2k dmg and shouldnt be like this.

![](http://i.imgur.com/HQyXqVx.jpg)

As a matter of fact stormstrike can trigger windfuries from both weapons in the same swing along with 2 white attacks
So in a perfect rng scenario of no miss and all crit an enhancement shaman can execute
1 stormstrike (2 attacks crit)+mh wf(2 attacks crit)+oh wf(2 attacks crit)+mh white+oh white

![](http://i.imgur.com/8pPxyaa.jpg?1)

SUGGESTION
 
1)There should be an event selector on successful melee swing weather it is windfury or common attack in order not to have white attacks and winfury in the same swing.
2)In the event of stromstrike where both weapons hit at the same time it seems to be no shared icd for weapons

**Comment by Delinquent1** [comment]
Mind adding a comment here saying you look in it or that you have no clue so we can help you?

**Comment by Fleischi** [comment]
mind adding a source that backs up your suggestions?
because this looks more like a wish list than a bug report if you do not prove that things were or were not that way on retail

**Comment by Delinquent1** [comment]
The only evidence i could provide is from archived posts sadly
http://forums.elitistjerks.com/forums/topic/41711-enh-windfury-weapon-cooldown/?page=1
If you go through this you will see that some people claim to see it and others do not
Some conclude into that its clientside issue of tracking timestamps in the combat log affected mostly by latency and one should not rely on that

**Comment by corcaroli** [comment]
I'm not sure if i should open a new issue, so i'll post here.
I usually play as elemental, but in the last month i had some opportunities to play as enhancement. And what i noticed during that were weird quadruple windfury swings instead of usual double:
http://i.imgur.com/lnie03A.jpg
http://i.imgur.com/sVKFXNF.jpg
http://i.imgur.com/5I3jL0E.jpg
Another thing i noticed was that sometimes after a windfury proc the next one would come in less than 3 seconds. I am not absolutely sure about that, because i weren't focused on watching the windfury cooldown during the boss encounters, but at the same time don't think it was my imagination. I will definitely try to check this later.

**Comment by Dorakrab** [comment]
already reported in #3777 but noone seems to notics this

**Comment by corcaroli** [comment]
No, #3777 is about windfury totem issue, not windfury weapon imbue.

**Comment by corcaroli** [comment]
Here's the evidence of the internal windfury imbue cooldown:
http://sendvid.com/h3u239xf
The red moving bar slightly above the action bars tracks the remaining windfury cooldown (it's an addon called DisqoDice).
As you can see on video intervals 0:12-0:16, 0:18-0:25, 0:47-0:50, the internal windfury imbue cooldown is definitely less than 3 seconds.
Also you can see one of those weird quadruple windfury swings at 1:03.
(These actually might be simultaneous windfury procs from both main and off hand weapons, but that's only an assumption)

**Comment by Dorakrab** [comment]
sorry my bad, didnt read your post right

**Comment by Zerixx** [comment]
Fixed this. WF Weapon has a 3 sec ICD but the totem proc does not. Should only proc off white hits.. not yellow unless it's future cast like HS or Cleave. This is fixed 85% of the way.. just gotta ween it off of yellow hits for now.
